index.html

```diff
@@ -15,13 +15,17 @@
     <link rel="stylesheet" href="css/modules.css">
     <!-- jQ 3.1.1 -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <!-- Handlebars Template -->
     <script id="project-template" type="text/x-handlebars-template">
-      <p>{{title}}</p>
+      <article data-project="{{title}}" id="{{title}}">
+      <p>{{title}}</p
```

Comment:
I would suggest adding more than just p tags. I would make your title an h2 and contributors an h5 while class and week can be h4 and url can stay a p tag . (This is just an example)